### PR TITLE
[ES] Main, Introduction, Credits

### PR DESCRIPTION
**Pull request by luischos**
Credits and Introduction
There is a new image to title page, is not definitive, just create the image

credits
intro
introduction
title

### CODE REVIEW

**Comment by luischos** [comment]
@qwrtln is it posible you could create the spanish image for the title page? I don't have the font style.

**Comment by qwrtln** [comment]
Yes, please paste the text here.

File: translations/credits.tex/es.po [hunk 1]
```diff
@@ -32,6 +32,8 @@ msgid ""
 "{2}\n"
 "\n"
 msgstr ""
+"{2}\n"
+"\n"
```
Comment (qwrtln) [inline translations/credits.tex/es.po:36]:
```suggestion
```
Don't translate those without any text to translate. `po4a` will pick up the original automatically.

File: translations/credits.tex/es.po [hunk 2]
```diff
@@ -140,6 +162,8 @@ msgid ""
 "\\phantom{Translators placeholder}\n"
 "\n"
 msgstr ""
+"\\phantom{Translators placeholder}\n"
+"\n"
```
Comment (qwrtln) [inline translations/credits.tex/es.po:166]: This is the place where you type "Translation to Spanish: ..."

File: translations/introduction.tex/es.po [hunk 3]
```diff
@@ -37,6 +37,11 @@ msgid ""
 "several different Factions, led by their Heroes, battle for supremacy. "
 "Choose your Faction and Hero and banish your unruly enemies from these lands!"
 msgstr ""
+"\\hypertarget{Heroes of Might and Magic III}{\\textbf{Heroes of Might and "
+"Magic III: El juego de mesa}} es un juego de mesa RPG de estrategia táctica para 1-3 "
```
Comment (qwrtln) [inline translations/introduction.tex/es.po:41]:
```suggestion
"Magic III: The Board Game}} es un juego de mesa RPG de estrategia táctica para 1-3 "
```
I suggest leaving the name of the original. You have the translation right next to it anyway.

File: main_es.tex [hunk 4]
```diff
@@ -12,11 +12,11 @@
 }
```
Comment (qwrtln) [inline main_es.tex:12]: Just to be clear: "Version" in Spanish is also "Version"?

File: make_printable_es.sh [hunk 5]
```diff
@@ -0,0 +1 @@
+po4a --no-update po4a.cfg && latexmk -shell-escape -pdf main_es
```
Comment (qwrtln) [inline make_printable_es.sh:1]: This is not the purpose of this script. Please make an alias on your local machine and delete this file from the repo.

File: translations/credits.tex/es.po (hunk 1 above)
Comment (luischos) [reply translations/credits.tex/es.po:36]: The text: MANUAL DE REGLAS REESCRITO

File: translations/introduction.tex/es.po (hunk 3 above)
Comment (luischos) [reply translations/introduction.tex/es.po:41]: I think the same, but I was not sure

File: translations/credits.tex/es.po (hunk 1 above)
Comment (qwrtln) [reply translations/credits.tex/es.po:36]: Pushed!

File: main_es.tex (hunk 4 above)
Comment (luischos) [reply main_es.tex:12]: Corrected

**Comment by luischos** [comment]
I think all are solved

**Comment by qwrtln** [comment]
There is one more thing to translate. Please go to the `.github/workflows/build-and-publish-rule-book.yaml`, and add Spanish with translations to the matrix.

File: translations/credits.tex/es.po (hunk 1 above)
Comment (qwrtln) [reply translations/credits.tex/es.po:36]: Please delete those translations with no actual text.

File: translations/credits.tex/es.po [hunk 6]
```diff
@@ -140,6 +162,8 @@ msgid ""
 "\\phantom{Translators placeholder}\n"
 "\n"
 msgstr ""
+"\\phantom{Traducción al español: Luis Simón}\n"
```
Comment (qwrtln) [inline translations/credits.tex/es.po:165]:
```suggestion
"\\textbf{Traducción al español}: Luis Simón\n"
```
Phantom makes the text invisible ;-)

File: .github/workflows/build-and-publish-rule-book.yaml [hunk 7]
```diff
@@ -32,6 +32,9 @@ jobs:
           - language: pl
             version: Wersja
             warning: "To jest nieopublikowana wersja zbudowana na GitHubie z commita"
+          - language: pl
+            version: Wersja
```
Comment (qwrtln) [inline .github/workflows/build-and-publish-rule-book.yaml:36]:
```suggestion
          - language: es
            version: Versión
```

Comment (qwrtln) [reply .github/workflows/build-and-publish-rule-book.yaml:36]: Commit that one, too.

File: translations/introduction.tex/es.po [hunk 8]
```diff
@@ -37,6 +37,11 @@ msgid ""
 "several different Factions, led by their Heroes, battle for supremacy. "
 "Choose your Faction and Hero and banish your unruly enemies from these lands!"
 msgstr ""
+"\\hypertarget{Heroes of Might and Magic III}{\\textbf{Heroes of Might and "
+"Magic III: The Board Game}} es un juego de mesa RPG de estrategia táctica para 1-3 "
+"jugadores usando el juego base.  El continente de Antagarich está en guerra ya que "
```
Comment (qwrtln) [inline translations/introduction.tex/es.po:42]:
```suggestion
"jugadores usando el juego base. El continente de Antagarich está en guerra ya que "
```

**Comment by qwrtln** [comment]
And finally, in the first comment please post pictures of the translated pages.

File: .github/workflows/build-and-publish-rule-book.yaml (hunk 7 above)
Comment (luischos) [reply .github/workflows/build-and-publish-rule-book.yaml]: Yes, done

File: translations/credits.tex/es.po [hunk 9]
```diff
@@ -45,6 +47,8 @@ msgid ""
 "\\vspace*{\\fill}\n"
 "\n"
 msgstr ""
+"\\vspace*{\\fill}\n"
+"\n"
```
Comment (qwrtln) [inline translations/credits.tex/es.po:51]:
```suggestion
```

File: translations/credits.tex/es.po (hunk 1 above)
Comment (qwrtln) [inline translations/credits.tex/es.po:36]:
```suggestion
```

**Comment by qwrtln** [comment]
The PDF's layout is broken because the text in section headers is move down due to Spanish accents. I'll try to fix this.

**Comment by luischos** [comment]
@qwrtln all ok from my side? anything else?

**Comment by qwrtln** [comment]
> @qwrtln all ok from my side? anything else?

Yes, we need to fix the text move downwards in section headings due to accents.

**Review by qwrtln — APPROVED** [review]

**Pull request merged**